apps/analytics/serializers.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+
+
+class SubmissionStats(object):
```

Comment:
I am actually not sure if this will go in master or not. So will update only if it needs to be merged. I hope that is fine.